### PR TITLE
[PE-389-1][PE-1517] chore: remove the domain prop

### PR DESCRIPTION
**Pull request by luqven**
This commit removes the `domain` prop from the `Imgix` component to fix an issue with the types. The `@types/react-imgix` repo doesn't yet have the new `domain` prop defined.

### Stacked PR Chain: PE-389-1
| PR | Title |  Merges Into  |
|:--:|:------|:-------------:|
|#49|[PE-389-1][PE-1531] feat: add loading state to asset grid|**N/A**|
|#51|[PE-389-1][PE-1517] chore: remove the domain prop|#49|
|#52|[PE-389-1][PE-1527] feat: add search to asset browser|#51|
|#53|[PE-389-1][PE-1528] feat: catch and display imgixAPI errors|#52|
|#54|[PE-389-1][PE-1526] feat: add pagination|#53|
|#55|[PE-389-1][PE-1526-1] chore: update grid style to show more assets|#54|

### CODE REVIEW

File: frontend/src/components/grids/AssetGrid.tsx [hunk 1]
```diff
@@ -14,8 +14,7 @@ export function AssetGrid({ assets, domain }: Props): ReactElement {
       <div className="ix-grid-item" key={`${asset.id}-${idx}`}>
         <div className="ix-grid-item-image">
           <Imgix
-            domain={domain}
-            src={asset.attributes.origin_path}
+            src={"https://" + domain + asset.attributes.origin_path}
```
Comment (frederickfogerty) [inline frontend/src/components/grids/AssetGrid.tsx:17]: I assume so, but just checking: does `asset.attributes.origin_path` always start with a `/`?

**Review by frederickfogerty — APPROVED** [review]
LGTM

File: frontend/src/components/grids/AssetGrid.tsx (hunk 1 above)
Comment (luqven) [reply frontend/src/components/grids/AssetGrid.tsx:17]: AFAIK yes. I've yet to run into an instance where that's not the case. We might want to "normalize" these paths in future to play it safe.

**Comment by sherwinski** [comment]
Just FYI, this should be working now https://github.com/DefinitelyTyped/DefinitelyTyped/commit/59ee82d210a8b9850e38302066544764566b36e7

**Pull request merged**